update UG (#48)
AY1920S1-CS2103T-T13-4/main: docs/UserGuide.adoc

then annotations are shown as well.

=== Adding a reminder: `reminder`

Adds a reminder to visit a bookmark. You can set the time for the reminder.

Format: `reminder INDEX t/TIME_DATE [n/NOTE=Open]`

* INDEX is the index of the bookmark that you want to add with a reminder.
 * TIME_DATE follows HHMM dd/mm/yyyy format.
* NOTE is the description of the reminder, the default value is Open.

Examples:

* `reminder 1 t/2359 02/07/2019` +
Adds a reminder for bookmark 1, and sets the time of the reminder as 23:59 of 02/07/2019.
It reminds you to open the bookmark.

* `reminder 1 t/2359 02/07/2019 n/Read announcements` +
Adds a reminder for bookmark 1, and sets the time of the reminder as 23:59 of 02/07/2019.
The reminder reminds you to read announcements.

=== Editing a reminder: `reminder edit`

> Edits the reminder of the specified bookmark. You can edit either time or the note of the reminder.

Reviewer: ```suggestion
Edits the reminder of the specified bookmark. You can edit either the time or the note of the reminder.
```

Format: `reminder edit INDEX [t/TIME_DATE] [n/NOTE]`

> * INDEX is the index of the bookmark that has the reminder that you want to edit.

Reviewer: Following the discussion on Telegram: `INDEX` should be the index of the reminder, correct?

Author: Okay

then annotations are shown as well.

=== Adding a reminder: `reminder`

Adds a reminder to visit a bookmark. You can set the time for the reminder.
You can also choose when to open the bookmark.

> The default setting is open.

Reviewer: Can I check, is `the time for the reminder` (line 545) the same as `when to open the bookmark` (line 546)? It looks like you can set two different times. Also, suggested change below:
```suggestion
```

Author: Fixed. Can you check again the description and the example?

> === Adding reminder: `reminder`

Reviewer 2: `Adding a reminder`

Author: Fixed.

Adds a reminder to visit a bookmark. You can set the time for the reminder.
You can also choose whether to open the bookmark.

> The default setting is open.

Reviewer 2: Do we have this setting?

Author: It changes to a Nice to Have feature.

Author: I'll edit it.

Format: `reminder INDEX t/TIME_DATE [n/NOTE=Open]`

* INDEX is the index of the bookmark that you want to add with a reminder.
 * TIME_DATE follows HHMM dd/mm/yyyy format.
* NOTE is the description of the reminder, the default value is Open.

Examples:

* `reminder 1 t/2359 02/07/2019` +
Adds a reminder for bookmark 1, and sets the time of the reminder as 23:59 of 02/07/2019.
It reminds you to open the bookmark.

* `reminder 1 t/2359 02/07/2019 n/Read announcements` +
Adds a reminder for bookmark 1, and sets the time of the reminder as 23:59 of 02/07/2019.
The reminder reminds you to read announcements.

> === Editing reminder: `reminder edit`

Reviewer 2: Suggestion: `Editing a reminder`

Author: Fixed.

> Edit a reminder at the reminder's index. You can edit either time or whether to open it.

Reviewer 2: `Edits`

Format: `reminder edit INDEX [t/TIME_DATE] [n/NOTE]`
* INDEX is the index of the bookmark that has the reminder that you want to edit.
* TIME_DATE is the time you want to edit. The time follows HHMM dd/mm/yyyy format.
* NOTE is the description of the reminder that you want to edit.
* `reminder edit 1 t/2359 02/07/2019` +

> Edit reminder of bookmark in index 1. Reset the time of the reminder to 23:59 of 02/07/2019.

Reviewer 2: ```suggestion
Edits the reminder of bookmark 1. Reset the time of the reminder to 23:59 of 02/07/2019.
```

Reviewer: @TSAI-HSIAO-HAN 

FYI, you can commit all of the suggested changes directly from GitHub:
1. Go to the tab `Files Changed`
2. Click `Add suggestion to batch` for each suggested change. 
3. Click `Commit` (or whatever the button is named) once you're done adding all the changes.

* `reminder 1 t/2359 02/07/2019 n/Check daily news` +
Edit reminder of bookmark in index 1. Reset the time of the reminder to 23:59 of 02/07/2019.
Reset to not to open the bookmark. Reset the note to check daily news.

> === Deleting reminder: `reminder delete`

Reviewer 2: `Deleting a reminder`

Author: Fixed.

=== Adding reminder: `reminder`

Adds a reminder to visit a bookmark. You can set the time for the reminder.
You can also choose whether to open the bookmark.
The default setting is open.

Format: `reminder INDEX t/TIME_DATE [d/DESCRIPTION] [n/NOTE=Open]`

* INDEX is the index of the bookmark that you want to add with a reminder.
 * TIME_DATE follows HHMM dd/mm/yyyy format.
* DESCRIPTION is the description of the reminder.

> * NOTE is the setting of whether you want to open the bookmark.

Reviewer: If I remember correctly, we intended `NOTE` to be the description that is displayed in the reminder list. The text "Open" is the default text if there is no description. For example: "*Open bookmark 1 by 23/9/19 18:00*". (Refer to the dashboard mockup [here](https://github.com/AY1920S1-CS2103T-T13-4/main/blob/master/docs/images/ui-screenshots/uimockup1_dashboard.png).) 

Since `DESCRIPTION` now handles this role, **can I clarify what the function of `NOTE` is**? :) Does it automatically open a bookmarked webpage at the specific time? If yes, I'm concerned that it might be a bit intrusive for the user... Perhaps we could have a pop-up reminder at the specified time instead.

In either case, I think renaming the parameter `n/NOTE` would make it easier for the user to understand. For example, if the purpose of `n/NOTE` is modified, it could be renamed to `p/POP_UP`. (With the default value of `p/POP_UP` being `true`.)

Author: Okay, then I just use n/NOTE

Examples:

* `reminder 1 t/2359 02/07/2019` +

> Adds a reminder for bookmark 1, and set the time at 02/07/2019.

Reviewer: ```suggestion
Adds a reminder for bookmark 1, and sets the reminder time as 23:59 of 02/07/2019.
```
Just a suggestion, to make the verb tense consistent. (Similar comments for the other examples)

Author: Fixed.

It will also open the bookmark for you.

* `reminder 1 t/2359 02/07/2019 n/Close` +
Adds a reminder for bookmark 1, and set the time at 23:59 of 02/07/2019.

> The reminder would not open the bookmark for you.

Reviewer: Another suggestion, to improve clarity. May not be needed if the parameter is changed.
```suggestion
The reminder will not open the bookmark at the specified time.
```

Author: Changed the parameter.

* `tab 3` +
Goes to the Offline tab which shows the offline copy of bookmarks.

> === Autocomplete commands `[coming in v2.0]`

Reviewer 2: ```suggestion
=== Autocompleting partially typed commands `[coming in v2.0]`
```

Author: Fixed.

> Auto-completes search for commands line if the command is unique.

Reviewer 2: ```suggestion
Automatically fills in partially typed commands if the command is unique.
```

I am not sure about this but personally I feel such phrasing is clearer.

Author: Fixed

> === Share bookmarks and notes via Email/ Telegram `[coming in v2.0]`

Reviewer 2: ```suggestion
=== Sharing bookmarks and notes via Email/ Telegram `[coming in v2.0]`
```

Author: Fixed.

* `tab 3` +
Goes to the Offline tab which shows the offline copy of bookmarks.

=== Autocomplete commands `[coming in v2.0]`
Auto-completes search for commands line if the command is unique.

=== Share bookmarks and notes via Email/ Telegram `[coming in v2.0]`

> Shares your bookmarks and mots to your friends via Email or Telegram.

Reviewer: ```suggestion
Shares your bookmarks and notes with your friends via Email or Telegram.
```
Alternative:
```suggestion
Sends your bookmarks and notes to your friends via Email or Telegram.
```

Author: Use the first suggestion.
